templates/views/email-password-interaction/confirm.ejs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Authorize</title>
+    <link rel="stylesheet" href="/idp/style.css">
```

Comment:
What exactly, the fact that the "idp" string is hardcoded here? There's an issue open to change that still but I don't think that needs to be fixed before the merge.

Reply:
I already see `<%= details.pathPrefix %>` below now.

Reply:
That was already there in the original version. The problem is that nowhere in the code this is actually being filled in with an actual value 😅 . The reason I didn't add it for the stylesheet link is because some classes don't expose this details object (yet) so I just postponed all of that until the relevant issue is fixed.